database/migrations/2024_02_23_220158_add_column_to_belongings.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddColumnToBelongings extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('belongings', function (Blueprint $table) {
+            $table->unsignedBigInteger('added_by_id'); // Define user_id column
+            $table->foreign('added_by_id')->references('id')->on('users'); // Add foreign key constraint
```

Comment:
No need to create column then assign index. 
Do: `$table->foreignId('added_by_id)->nullable()->constrained('users')->onDelete(`set null')->onUpdate('set null);

nullable: to allow null values
onDelete('set null'): sets null on delete event
onUpdate('set null'): sets null on update event

We do this to avoid conflicts with current data (if any) 
+ If user got deleted we don't want the data get affected